SpatialGDK/Documentation/content/game-client-interest-management.md

```diff
@@ -0,0 +1,115 @@
+
+Game clients need information about Actors that they don’t own (see the Unreal documentation on [owning](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/OwningConnections) to help them to correctly manipulate the Actors that they _do_ own, and to render the SpatialOS world. Interest enables game clients to receive the relevant information.
+
+You can define your game client’s interest in three ways, which you can use alongside each other:
+
+* **`NetCullDistanceSquared`**<br>
+A property that you define on an Actor’s class. For any Actor of that class or a derived class, game clients whose players are within the distance you specify receive updates about the Actor.<br>
+* **`ActorInterestComponent`**<br>
+An Unreal Actor component that you add to an Actor that a game client might own. This includes Actors that are possessed by Actors that the game client owns. For example, if a game client owns a PlayerController, and the PlayerController possesses a Pawn, then the game client owns the Pawn.
+
+    Within this component, you provide a list of queries that express the Actors that you want the game client to receive updates about. This is a more granular way of setting up interest than `NetCullDistanceSquared`.<br>
+* **`AlwaysInterested`**<br>
+A `UPROPERTY` that you add to an Actor that a game client might own. It refers to a specific Actor that you want the game client to always receive updates about.
+
+> **Tip**: All three ways of defining interest can coexist. However, if you've specified all of the interest in your game using `ActorInterestComponent` and `AlwaysInterested`, you can turn off `NetCullDistanceSquared` using the toggle in `ActorInterestComponent`.
+
+## NetCullDistanceSquared
+You can define a distance using `NetCullDistanceSquared` on an Actor class. Game clients whose players are within this distance of an Actor of this class receive updates about the Actor. By default, this distance is 150 meters for all Actor classes.
+
+To change an Actor class’s `NetCullDistanceSquared` value, open the Actor class's Blueprint and go to the Class Defaults tab. Net Cull Distance Squared is in the Replication section:
+
+![NetCullDistanceSquared in the Replication section]({{assetRoot}}assets/screen-grabs/game-client-interest-management/replication-section.png)
+
+> **Tip**: You need to convert your chosen number of meters into centimeters, square it, and then enter the resulting value. For example, to set a distance of 150 meters, you need to enter 225000000 (like in the screenshot above).
+
+### Example 
+If you set the `NetCullDistanceSquared` to 200 meters for all Characters, game clients receive updates about Characters that are within 200 meters of their own player.
+
+And if you set the `NetCullDistanceSquared` to 400 meters for vehicles, game clients receive updates about vehicles within 400 meters of their player.
```

Comment:
Potentially we might want to make a concrete example here as well.

Reply:
Yep, I was struggling a bit with things like "vehicle" and "character" and whether we were using them as normal English words, or whether they were Unreal keywords that referred to an Actor class that always existed, or whatever. (Come to think of it, is an "Actor Blueprint" the same as an Actor class?)

For this part of the example, would it be something like:

> If you also have an Actor Blueprint called `Vehicle`, and you set its `NetCullDistanceSquared` to 400 meters, then game clients will receive updates about all `Vehicle`s within 400 meters of their player.

Reply:
That works well